lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -795,6 +796,65 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
     """
   end
 
+  @doc """
+  Chart component
+
+  You can see it in use in Metrics page.
+  """
+
+  attr :id, :string,
+    required: true,
+    doc: "Because is a stateful `Phoenix.LiveComponent` an unique id is needed."
+
+  attr :data, :list,
+    default: [],
+    doc: """
+    Temporary list of points to show in the chart.
+    Each element should be the format `{optional_label, x, y}`.
+    New points can be added using the function `send_data_to_chart/2` in real time.
+    """
+
+  attr :title, :string, required: true, doc: "Title of the chart"
+  attr :hint, :string, default: nil, doc: "A textual hint to show close to the title."
+
+  attr :kind, :atom,
+    values: [:counter, :last_value, :sum, :summary, :distribution],
+    doc: "Kind of chart to use."
```

Comment:
This is still very coupled to telemetry metrics system so I don't like it. We could set it to: `[:line, :summary, :histogram]` but then we have to move the following functionality to the server side: https://github.com/phoenixframework/phoenix_live_dashboard/blob/master/assets/js/metrics_live/index.js#L131-L137

What do you think?